lib/date.js

```diff
@@ -0,0 +1,57 @@
+$('document').ready(function(){
+  var todayDate = moment();
+  $(".current-date").append(todayDate.format("MMMM Do, YYYY"));
+  localStorage.setItem("currentDate", JSON.stringify(todayDate.format("MMMM Do, YYYY")));
+  currentDate = todayDate;
+  add_day();
+
+  $(".day-foreward").on("click", function(){
+    currentDate = currentDate.add(1, 'days');
+    localStorage.setItem("currentDate", JSON.stringify(currentDate.format("MMMM Do, YYYY")));
+    $(".current-date").html(currentDate.format("MMMM Do, YYYY"));
+    add_day();
+  });
+
+  $(".day-backward").on("click", function(){
```

Comment:
this event handler and the event handler attached to ".day-forward" look pretty similar. a good place for some refactoring (eventually) 👍